src/relay.cpp

```diff
@@ -754,12 +755,13 @@ void callback_dual_tor(evutil_socket_t fd, short event, void *arg) {
         current_position = tmp;
 
         if (current_position + sizeof(struct dhcpv6_msg) > ((uint8_t *)ptr + buffer_sz)) {
-            syslog(LOG_WARNING, "Invalid DHCPv6 packet length %d, no space for dhcpv6 msg header\n", buffer_sz);
+            syslog(LOG_WARNING, "Invalid DHCPv6 packet length %ld, no space for dhcpv6 msg header\n", buffer_sz);
```

Comment:
%d should ok here for buffer_sz as it will not over BUFFER_SIZE.
For more correct way for size_t variable here may “%zu”.

Reply:
%d gives a warning. Updated to %zu